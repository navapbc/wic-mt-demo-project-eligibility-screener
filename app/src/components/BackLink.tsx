import { useTranslation } from 'next-i18next'
import { ReactElement } from 'react'

import StyledLink from '@components/StyledLink'

export interface BackLinkProps {
  href: string
}

export const BackLink = (props: BackLinkProps): ReactElement => {
  const { href } = props

  const { t } = useTranslation('common')
  return <StyledLink href={href} text={t('backlinkText')} />
}

export default BackLink
