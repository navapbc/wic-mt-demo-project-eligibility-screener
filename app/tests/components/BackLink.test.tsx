import { render, screen } from '@testing-library/react'

import BackLink from '@components/BackLink'

import { testSnapshot } from '../helpers/sharedTests'

it('should match snapshot', () => {
  testSnapshot(<BackLink href="/" />)
})
