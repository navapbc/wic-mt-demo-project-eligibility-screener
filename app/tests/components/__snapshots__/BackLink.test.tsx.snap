// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<div>
  <a
    class="usa-link"
    href="/"
  >
    Back
  </a>
</div>
`;
